openapi/components/schemas/RichBillingPortal.yaml

```
@@ -0,0 +1,89 @@
+type: object
+required:
+  - websiteId
+  - name
+  - plans
+properties:
+  id:
+    type: string
+    description: ID of the billing portal.
+    readOnly: true
+    maxLength: 50
+    example: bill_prt_0YV7K5TYV5D1P9ZNKDT39KZC3C
+  websiteId:
+    $ref: ./WebsiteId.yaml
+  name:
+    type: string
+    description: Name of the billing portal.
+  status:
+    description: Status of the billing portal.
+    type: string
+    enum:
+      - active
+      - inactive
+    default: active
+  plans:
+    type: array
+    minItems: 1
+    items:
+      type: object
+      properties:
+        plan:
+          allOf:
+            - type: object
+              description: Plan details.
+              required:
+                - id
+              properties:
+                id:
+                  type: string
+                  description: ID of the plan.
+                  maxLength: 50
+                  example: plan_0YV7DENSVGDBW9S71XZNNYYQ0X
+            - $ref: ./SubscriptionOrderPlan.yaml
+        status:
+          type: string
+          description: |-
+            Status of the plan in the billing portal.
+            If status is `archived` customer can continue using it, but can not select it as a new item.
```

suggestion: for readability
```suggestion
            If status the is `archived`, customers can continue using the billing portal, 
            but can not select it as a new item.
```